student/models.py

```diff
@@ -1,3 +1,30 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Gender(models.TextChoices):
+    MALE = "M", "Male"
+    FEMALE = "F", "Female"
+    UNSPECIFIED = "U", "Unspecified"
+
+
+class EducationalInstitution(models.Model):
+    institution_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.institution_name
+
+
+class Student(models.Model):
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, primary_key=True)
+    first_name = models.CharField(max_length=255)
```

Comment:
https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
Read it, still think sticking with a CharField will be alright.. if you think otherwise lmk..

Reply:
My comment was with regards to the separation between first and last name, why do you need to separate them?

Reply:
Not seeing updates in the code here yet